### DISCUSSION TITLE
Problem with navbar toggler

### DISCUSSION THREAD

**Discussion by KangaroujacK** [twbs/bootstrap#38396] [Q&A] [answered]
Hi there, it's my first time pointing to some sw problems for bootstrap, but even in general, so if I've mistaken the place just let me know and I'll change the place of the post. 

Right now I'm following the Udemy course "Web development bootcamp 2023" of Angela Yu. During "lesson 66: navigation bar" it happens an error once I'm trying to use the 'hamburger collapse' for the navbar, but I've seen that it's more common than it should be. 

I tried to take the plain code for the toggler navbar directly from bootstrap but it seems to have some similar issues.
Here ---> https://getbootstrap.com/docs/5.3/components/navbar/#toggler

  - once clicked the 'menu' icon it expands onto the navbar list but once clicked again it won't collapse to the icon.

Plus 
  - it happens something similar with the boostrap site. If you have a tablet-size window and you try to click the 'search' icon it won't shows. It will be shown only if the screen is enlarged within a laptop-size window. 

From my lack of knowledge I suppose it's something related to javascript but ... I don't know.

Kind regards, 
K.

**Comment by julien-deramond** [comment]
Hi @KangaroujacK.
Tried to reproduce the issue but I'm not sure to catch what's the problem, seems to work well with the current docs.
Could you maybe provide a .gif or something to see what's the actual issue?
With which browser/OS combination?

**Reply by KangaroujacK** [reply]
Sorry for the delay but I didn't see the mail with your answer. I attached the .gif file with the video demonstration of the problem.
![Bootstrap_problem_navbar_show](https://user-images.githubusercontent.com/121972123/235784531-ce39ea6d-e8a6-4b4b-a341-c5d705ef7105.gif)

I suppose that the problem it has been solved because know it will not happen anymore. 
BTW thanks for the response.

**Comment by JosephShown** [comment, marked as answer]
This could be due to a variety of reasons, including missing JavaScript dependencies or conflicts with other scripts on your page.

Make sure you have included the necessary JavaScript files for Bootstrap to function properly. You can check the Bootstrap documentation(https://getbootstrap.com/docs/5.3/getting-started/introduction/#js) for more information on how to include these files. 

I would open dev tools and go to the network tab and click clear and record then reload the page, along the row of options select js and look to see if your js file to handle this is loading. Most of the time i see this it is just a file location that is off.

**Reply by KangaroujacK** [reply]
Hi sorry for the late response, I tried your suggestion but I think that for some reason it's not working. I hopped again to my project today and I'll see if I can found what's the issue. 

Thanks.

**Comment by abrahamcahyoadi** [comment]
you can try Code navbar toggler:

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <nav class="navbar navbar-expand-lg bg-body-tertiary">
        <div class="container-fluid">
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo01" aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
          </button>
          <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
            <a class="navbar-brand" href="#">Hidden brand</a>
            <ul class="navbar-nav me-auto mb-2 mb-lg-0">
              <li class="nav-item">
                <a class="nav-link active" aria-current="page" href="#">Home</a>
              </li>
              <li class="nav-item">
                <a class="nav-link" href="#">Link</a>
              </li>
              <li class="nav-item">
                <a class="nav-link disabled">Disabled</a>
              </li>
            </ul>
            <form class="d-flex" role="search">
              <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
              <button class="btn btn-outline-success" type="submit">Search</button>
            </form>
          </div>
        </div>
      </nav>
      <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.min.js" integrity="sha384-Y4oOpwW3duJdCWv5ly8SCFYWqFDsfob/3GkgExXKV4idmbt98QcxXYs9UoXAB7BZ" crossorigin="anonymous"></script>

</body>
</html>
```

**Reply by KangaroujacK** [reply]
I don't know why but now it seems to work. I think that I've put the wrong links related to bootstrap in the head, or at least this is the only expaination that I can give to myself (it's the only thing that has/I changed). Though, I don't really know how this would be possible ...

Said that, thanks for trying helping me! :)
K.
